src/components/pages/Wallet.tsx

```diff
@@ -140,16 +141,14 @@ const Invite: React.StatelessComponent<Props> = props => {
 
   return (
     <React.Fragment>
-      <Text style={[styles.inviteSectionText]}>
-        You have nothing to mint at this time.
-      </Text>
+      <Text style={[styles.inviteSectionText]}>Nothing to mint right now.</Text>
       <View>
         <Text style={styles.inviteSectionText}>
-          Invite a friend to earn 60 Raha:
+          Invite a friend to create {REFERRAL_BONUS} Raha!
         </Text>
         <Button
           style={styles.button}
-          title="Invite"
+          title={`INVITE +${REFERRAL_BONUS}`}
```

Comment:
You don't need to capitalize it, use normal capitalization: Invite +${REFERRAL_BONUS}